Brokerages/Fxcm/FxcmBrokerage.cs

```diff
@@ -115,6 +116,11 @@ public FxcmBrokerage(IOrderProvider orderProvider, ISecurityProvider securityPro
             _password = password;
             _accountId = accountId;
 
+            _subscriptionManager = new EventBasedDataQueueHandlerSubscriptionManager();
+            _subscriptionManager.SubscribeImpl += Subscribe;
+            _subscriptionManager.UnsubscribeImpl += Unsubscribe;
+            _subscriptionManager.GetChannelName += (t) => "quote";
```

Comment:
Seems that `GetChannelName` would be better as an argument for `EventBasedDataQueueHandlerSubscriptionManager` constructor, don't think it would work without it?

Reply:
agree, done